presto-docs/src/main/sphinx/release/release-0.206.rst

```diff
@@ -0,0 +1,56 @@
+=============
+Release 0.206
+=============
+
+General Changes
+---------------
+
+* Fix query processing error for certain queries when ``dictionary_aggregation`` is enabled.
+* Fix query plan for aggregations over a single group,
+  e.g. ``select c1, sum(c2) from t where c1 = 'foo' group by c1``
+* Fix average row size computation to account for non-null rows only
```

Comment:
`Fix average row size computation to account for non-null rows only`: This is vague, what is this about? This needs a period at the end.